src/index.js

```diff
@@ -1,3 +1,9 @@
+import './bootstrap.scss';
 import './style.scss';
+import GameView from './lib/View /GameView';
```

Comment:
That's nice fuckup in path :D you have spacebar in folder name - and it works because the problem is in the name, not in the import :D